launcher/minecraft/mod/tasks/LocalModParseTask.cpp

```diff
@@ -293,86 +293,90 @@ ModDetails ReadFabricModInfo(QByteArray contents)
+                                                       obj.value("url").toString(), obj.value("description").toString()));
                 }
-            } else if (license.isString()) {
-                details.licenses.append(ModLicense(license.toString()));
-            } else if (license.isObject()) {
-                auto obj = license.toObject();
-                details.licenses.append(ModLicense(obj.value("name").toString(), obj.value("id").toString(), obj.value("url").toString(),
-                                                   obj.value("description").toString()));
             }
-        }
 
-        if (modMetadata.contains("icon")) {
-            auto icon = modMetadata.value("icon");
-            if (icon.isObject()) {
-                auto obj = icon.toObject();
-                // take the largest icon
-                int largest = 0;
-                for (auto key : obj.keys()) {
-                    auto size = key.split('x').first().toInt();
-                    if (size > largest) {
-                        largest = size;
+            if (modMetadata.contains("icon")) {
+                auto icon = modMetadata.value("icon");
+                if (icon.isObject()) {
+                    auto obj = icon.toObject();
+                    // take the largest icon
+                    int largest = 0;
+                    for (auto key : obj.keys()) {
+                        auto size = key.split('x').first().toInt();
+                        if (size > largest) {
+                            largest = size;
+                        }
                     }
-                }
-                if (largest > 0) {
-                    auto key = QString::number(largest) + "x" + QString::number(largest);
-                    details.icon_file = obj.value(key).toString();
-                } else {  // parsing the sizes failed
-                    // take the first
-                    for (auto i : obj) {
-                        details.icon_file = i.toString();
-                        break;
+                    if (largest > 0) {
+                        auto key = QString::number(largest) + "x" + QString::number(largest);
+                        details.icon_file = obj.value(key).toString();
+                    } else {  // parsing the sizes failed
+                        // take the first
+                        for (auto i : obj) {
+                            details.icon_file = i.toString();
+                            break;
+                        }
                     }
+                } else if (icon.isString()) {
+                    details.icon_file = icon.toString();
                 }
-            } else if (icon.isString()) {
-                details.icon_file = icon.toString();
             }
         }
+
+    } catch (const Exception& e) {
+        qWarning() << "Unable to parse mof info:" << e.cause();
```

Comment:
`mod`?
nit: a space might be nice to have after `:`

Reply:
```suggestion
        qWarning() << "Unable to parse mod info:" << e.cause();
```

Reply:
The space is added by the << operator